mitosheet/src/mito/components/modals/StreamlitSignupModal.tsx

```diff
@@ -0,0 +1,200 @@
+import TextButton from '../elements/TextButton';
+import { TaskpaneType } from '../taskpanes/taskpanes';
+
+const StepOne = (
+    props: {
+        next: () => void;
+        mitoAPI: MitoAPI;
+    }): JSX.Element => {
+
+    const [email, setEmail] = useState('');
+
+    
+    const onSubmit = async (e: FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        await props.mitoAPI.updateSignUp(email);
+        props.next();
+    }
+
+    return (
+        <div className='signup-modal-left-column'>
+            <div>
+                <h1 className='text-header-1' style={{margin: 0}}>
+                    Sign up for Mito
+                </h1>
+                <p className='text-header-2 text-color-medium-important' style={{marginTop: '10px'}}>
+                    We’ll send you periodic product updates and welcome any feedback.
+                </p>
+                <p className='text-header-2 text-color-medium-important' style={{marginTop: '10px'}}>
+                    This modal appears the first time you use Mito for Streamlit.
+                </p>
+            </div>
+            <form className='signup-modal-email-form' onSubmit={onSubmit}>
+                <p className='text-header-2' style={{marginTop: '10px', marginBottom: 0}}>
+                    Email
+                </p>
+                <Input
+                    value={email}
+                    onChange={(event) => {setEmail(event.target.value)}}
+                    type='email'
+                    width='large'
+                    placeholder='example@gmail.com' 
+                    required
+                    autoFocus
+                />
+                <div className='mt-10px' style={{marginTop: '10px'}}>
+                    <TextButton
+                        variant='dark'
+                        width='block'
+                        type='submit'
+                    >
+                        Accept
+                    </TextButton>
+                </div>
+            </form>
+        </div>
+    )
+}
+
+
+
+/* Step three asks the user want they want to use Mito for */
+const StepTwo = (
+    props: {
+        back: () => void;
+        next: () => void;
+        mitoAPI: MitoAPI;
+        isPro: boolean;
+    }): JSX.Element => {
+
+    return (
+        <div className='signup-modal-left-column'>
+            <div>
+                <h1 className='text-header-1' style={{marginTop: 0, marginBottom: 0}}>
+                    {props.isPro && `Mito Pro is Totally Private`}
+                    {!props.isPro && `Built for Privacy`}
+                </h1>
+                <p className='text-header-2' style={{marginTop: '10px'}}>
+                    {props.isPro && `As a Mito Pro user, no data leaves your computer, ever. Check out our CCPA compliant privacy policy`}
+                    {!props.isPro && `We make sure none of your private data leaves your computer in our basic telemetry. Read our CCPA compliant`}
```

Comment:
"We make sure none of your private data leaves your computer in our basic telemetry. " -> "We make sure none of your private data leaves your computer. We only collect information about how you use Mito, not your data."